samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs

```diff
@@ -12,6 +12,7 @@
 using Microsoft.SemanticKernel.Connectors.AI.OpenAI.TextEmbedding;
 using Microsoft.SemanticKernel.Connectors.Memory.AzureCognitiveSearch;
 using Microsoft.SemanticKernel.Connectors.Memory.Qdrant;
+using Microsoft.SemanticKernel.Connectors.Memory.Weaviate;
```

Comment:
Won't be possible to do this until the nuget package is available. Probably best to do as a separate PR.